src/org/jitsi/impl/neomedia/VideoMediaStreamImpl.java

```diff
@@ -440,10 +440,15 @@ else if(closestAspect == null)
         = new RTCPReceiverFeedbackTermination(this);
 
     /**
+     *
+     */
+    private final PaddingTermination paddingTermination = new PaddingTermination();
+
+    /**
      * The <tt>RemoteBitrateEstimator</tt> which computes bitrate estimates for
      * the incoming RTP streams.
      */
-    private final RemoteBitrateEstimator remoteBitrateEstimator
+    private final RemoteBitrateEstimatorSingleStream remoteBitrateEstimator
```

Comment:
Yeah it seems to me that if `RemoteBitrateEstimator`s end up implementing `TransformEngine` then it'd be good to denote that at the interface level.